main.go

```diff
@@ -56,23 +57,25 @@ type TestCase struct {
 
 func executeTestCase(testCase TestCase) {
 
-	// start with base url
 	callUrl, err := url.Parse(fmt.Sprintf("http://%v:8080/", testCase.Client))
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	// build query params
 	params := url.Values{}
 	params.Add("server", testCase.Server)
 	params.Add("behavior", testCase.Behavior)
 	callUrl.RawQuery = params.Encode()
 
-	// make request to test client
 	resp, err := http.Get(callUrl.String())
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	log.Println(resp.StatusCode, callUrl.String())
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	fmt.Println(fmt.Sprintf("%v - %v - %v", resp.StatusCode, callUrl.String(), string(body)))
```

Comment:
Use Printf.
Also, `%v` calls `String()` for you. Don't need to do it manually. Also, you can avoid doing `string(body)` by using `%s` for that.